sirepo/cookie.py

```diff
@@ -176,7 +176,8 @@ def _decrypt(self, value):
         return self._crypto().decrypt(base64.urlsafe_b64decode(value))
 
     def _deserialize(self, value):
-        v = value.split(_SERIALIZER_SEP)
+        v = value.decode("utf-8")
```

Comment:
I'd like to merge this pull request, but we need to remove the cookie.py changes per earlier discussion. Once you've removed these changes, we can merge and close SOW1.